File: lib/models/goal.dart
class GoalModel {
  final String? filePath;
  final int id;
  final dynamic weight, bmi, fatMass, muscleMass;
  void showInfo() {
    print("$id $weight $fatMass");
  }

  GoalModel.fromDB(Map<String, dynamic> db)
      : id = db['id'],
        weight = db['weight'],
        fatMass = db['fatMass'],
        bmi = db['bmi'],
        muscleMass = db['muscleMass'],
        filePath = db['filePath'];
}

File: models/goal.js
const Sequelize = require("sequelize");

const sequelize = require("../util/database");

const Goal = sequelize.define("goal", {
  id: {
    type: Sequelize.INTEGER,
    autoIncrement: true,
    allowNull: false,
    primaryKey: true,
  },
  weight: { type: Sequelize.DOUBLE },
  bmi: { type: Sequelize.DOUBLE },
  fatMass: { type: Sequelize.DOUBLE },
  muscleMass: { type: Sequelize.DOUBLE },
  userId: { type: Sequelize.INTEGER, allowNull: false },
});
module.exports = Goal;
